src/main/kotlin/no/nav/arbeidsgiver/altinnrettigheter/proxy/model/AltinnDataModels.kt

```diff
@@ -10,7 +10,7 @@ data class AltinnOrganisasjon(
         @JsonProperty("ParentOrganizationNumber")
         val parentOrganizationNumber: String? = null,
         @JsonProperty("OrganizationNumber")
-        val organizationNumber: String,
+        val organizationNumber: String?,
```

Comment:
Mulig å skrive en test på det?

Reply:
Ja, men jeg er litt usikker på om det er noe poeng :/ Det er jo et veldig enkelt tilfelle. Hvis noen fjerner det spørsmålstegnet, så antar jeg at de faktisk mener det de gjør, og det vil ikke hjelpe så mye med en "Er du sikker på at du vil gjøre dette?"-feilmelding fra testen... Men jeg er ikke helt sikker, hva tror du?

Reply:
Tenkte mer på noe som provoserer samme feil som vi fikk i PROD
`no.nav.arbeidsgiver.altinnrettigheter.proxy.altinn.AltinnException: Feil ved kall til Altinn
	at no.nav.arbeidsgiver.altinnrettigheter.proxy.altinn.AltinnClient.hentOrganisasjoner(AltinnClient.kt:59)
	at no.nav.arbeidsgiver.altinnrettigheter.proxy.service.AltinnrettigheterProxyService.hentOrganisasjoner(AltinnrettigheterProxyService.kt:18)`
[...]
`Caused by: org.springframework.web.client.RestClientException: Error while extracting response for type [java.util.List<? extends no.nav.arbeidsgiver.altinnrettigheter.proxy.model.AltinnOrganisasjon>] and content type [application/json]; nested exception is org.springframework.http.converter.HttpMessageNotReadableException: JSON parse error: Instantiation of [simple type, class no.nav.arbeidsgiver.altinnrettigheter.proxy.model.AltinnOrganisasjon] value failed for JSON property OrganizationNumber due to missing (therefore NULL) value for creator parameter organizationNumber which is a non-nullable type; nested exception is com.fasterxml.jackson.module.kotlin.MissingKotlinParameterException: Instantiation of [simple type, class no.nav.arbeidsgiver.altinnrettigheter.proxy.model.AltinnOrganisasjon] value failed for JSON property OrganizationNumber due to missing (therefore NULL) value for creator parameter organizationNumber which is a non-nullable type
 at [Source: (PushbackInputStream); line: 1, column: 256] (through reference chain: java.util.ArrayList[1]->no.nav.arbeidsgiver.altinnrettigheter.proxy.model.AltinnOrganisasjon[\"Organiza`

Reply:
Da blir det noe som en `ApiTest` med mock response fra Altinn for et enkelt foretak

Reply:
Jeg kan evt legge inn en bedrift uten orgnr i mockingen? Da vil ApiTest feile. Eller tenker du på enhetstesting?

Reply:
Det er sikkert nok med bedrift uten orgnr i mockingen